cfg/conf.sample.php

```diff
@@ -33,7 +33,7 @@
 ; (optional) set a syntax highlighting theme, as found in css/prettify/
 ; syntaxhighlightingtheme = "sons-of-obsidian"
 
-; size limit per paste or comment in bytes, defaults to 10 Mebibytes
+; size limit per paste or comment in bytes, defaults to 10 Megabytes
```

Comment:
I unfortunately have to say that the way it is called "Mebibytes" is correct.

These are MiB, not MB.
See https://en.wikipedia.org/wiki/Binary_prefix